cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/service/DatasetInstanceService.java

```diff
@@ -504,12 +503,10 @@ private void updateExplore(DatasetId datasetInstance, DatasetInstanceConfigurati
     try {
       exploreFacade.updateExploreDataset(datasetInstance.toId(), oldSpec, newSpec);
     } catch (Exception e) {
-      String msg = String.format("Cannot enable exploration of dataset instance %s of type %s: %s",
-                                 datasetInstance, creationProperties.getProperties(), e.getMessage());
-      LOG.error(msg, e);
+      LOG.error("Cannot update explore for dataset instance {} with old properties {} and new properties {}: {}",
```

Comment:
`e.getMessage` seems redundant, since we're passing `e` as the last parameter.
Won't the message get logged twice now?